README.md

```diff
@@ -104,10 +104,14 @@ There is no need to save manually.
 Address book data are saved in a file called `addressbook.txt` in the project root folder.
 You can change the location by specifying the file path as a program argument.
 
-Example: 
+Example:
 * `java seedu.addressbook.AddressBook mydata.txt`
+* `java seedu.addressbook.AddressBook myFolder/mydata.txt`
 
-> The file name must end in `.txt` for it to be acceptable to the program.
+> The file path must contain a valid file name and a valid parent directory.
+> File name is valid if it has an extension and no reserved characters(OS-dependent).
+> Parent directory is valid if the parent directory exists.
+> If a file already exists, it must be a regular file.
```

Comment:
If you look at the markdown rendering, this whole block is squished together in one line, which may not be what you were intending?